### PR TITLE
[zh] Sync glossary reference

### PR DESCRIPTION
**Pull request by tengqm**
related: #25247, #25244

### CODE REVIEW

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm

**Comment by tengqm** [comment]
@tanjunchen The deploy failure was real. One of the markdown file has incomplete front matter. Just fixed.
Thanks for review and approve again.

File: content/zh/docs/reference/glossary/control-plane.md [hunk 1]
```diff
@@ -47,9 +43,9 @@ tags:
 这个编排层是由多个不同的组件组成，例如以下（但不限于）几种：
 
  * {{< glossary_tooltip text="etcd" term_id="etcd" >}}
- * {{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}
- * {{< glossary_tooltip text="Scheduler" term_id="kube-scheduler" >}}
- * {{< glossary_tooltip text="Controller Manager" term_id="kube-controller-manager" >}}
- * {{< glossary_tooltip text="Cloud Controller Manager" term_id="cloud-controller-manager" >}}
+ * {{< glossary_tooltip text="API 服务器" term_id="kube-apiserver" >}}
```
Comment (helight) [inline content/zh/docs/reference/glossary/control-plane.md:46]: Server 这个不翻译吧

Comment (tengqm) [reply content/zh/docs/reference/glossary/control-plane.md:46]: 要翻译。

Comment (tanjunchen) [reply content/zh/docs/reference/glossary/control-plane.md:46]: see 
image

https://kubernetes.io/zh/docs/contribute/localization_zh/

**Pull request merged**